doc/operation_guides/manual/installation-guide.md

```diff
@@ -1,182 +1,171 @@
-# Installation of OpenProject 4.2 with Apache on Debian 7.7 or Ubuntu 14.04 LTS
+# Installation of OpenProject 4.2 with Apache on Ubuntu 14.04. LTS
 
-**This tutorial helps you to deploy OpenProject 4.2. Please, aware that:**
+This tutorial helps you to deploy OpenProject 4.2. Please, aware that:
 
-1. This guide requires that you have a clean **Debian 7.7 x64** or **Ubuntu 14.04 x64** installation with administrative rights. We have tested the installation guide on a Debian minimal netinstall image and on an Ubuntu Server image, but it should work on any derivative.
-2. OpenProject will be installed with a MySQL database (the guide should work analogous with PostgreSQL).
-3. OpenProject will be served in a production environment with Apache (this guide should work analogous with other servers, like nginx and others)
+This guide requires that you have a clean Ubuntu 14.04 x64 installation
+with administrative rights. We have tested the installation guide on a
+on an Ubuntu Server image, but it should work on any derivative.
 
-In this guide, we will install **OpenProject 4.2** with a **MySQL** database. Openproject will be served with the **Apache** web server. When your server needs to reboot, OpenProject should start automatically with your server.
+OpenProject will be installed with a MySQL database (the guide should
+work analogous with PostgreSQL).
```

Comment:
@jonasheinrich as far as I know both can be used (http://www.dict.cc/deutsch-englisch/analog.html).